packaging/test_irods_client_rest_cpp.py

```diff
@@ -475,3 +553,10 @@ def test_changing_passwords_is_supported__issue_99(self):
         self.user.assert_icommand(['iinit', old_password])
         self.user.assert_icommand(['ils', '-ld'], 'STDOUT', [self.user.session_collection])
 
+def main():
+    rest = TestClientRest()
+    res = rest.construct_meta_ops_for_target("do", "data_object")
+    print(res)
+
+if __name__ == "__main__":
+    main()
```

Comment:
We should probably remove this in favor of using the `unittest` module to run tests.